python/pyarrow/tests/test_csv.py

```diff
@@ -1972,6 +1972,33 @@ def test_write_csv_decimal(tmpdir, type_factory):
     assert out.column('col').cast(type) == table.column('col')
 
 
+def test_large_binary_write_to_csv(tmpdir):
+    data_size = int(1E6)
+    file_name = tmpdir / "fixedsize_"+str(data_size)+".csv"
```

Comment:
I'm not sure why this is parameterizing the file name, but in any case, should try to follow PEP 8 and put spaces around operators.

(also, you can use a f-string)